Sources/Client/Views/Play/GameView.swift

```diff
@@ -44,70 +44,74 @@ struct GameView: View {
   var body: some View {
     JoinServerAndThen(serverDescriptor, with: account) { client in
       WithRenderCoordinator(for: client) { renderCoordinator in
-        VStack {
-          switch state {
-            case .playing:
-              ZStack {
-                WithController(controller, listening: $inputCaptured) {
-                  if controllerOnly {
-                    gameView(client: client, renderCoordinator: renderCoordinator)
-                  } else {
-                    InputView(listening: $inputCaptured, cursorCaptured: !inGameMenuPresented && cursorCaptured) {
-                      gameView(client: client, renderCoordinator: renderCoordinator)
-                    }
-                    .onKeyPress { [weak client] key, characters in
-                      client?.press(key, characters)
-                    }
-                    .onKeyRelease { [weak client] key in
-                      client?.release(key)
+        if let renderCoord = renderCoordinator {
```

Comment:
minor, but you could probably reuse the renderCoordinator identifier here

Reply:
done!